consensus/docker-compose-files/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN mkdir -p /var/hyperledger/db
 RUN mkdir -p /var/hyperledger/production
 WORKDIR $GOPATH/src/github.com/hyperledger/fabric/
 COPY . .
+WORKDIR $GOPATH/src/github.com/hyperledger/fabric/peer
+RUN pwd
```

Comment:
what is this for? debugging? Seems like if we want to have debugging, we should make it conditional, and probably not the default.